numpy/lib/nanfunctions.py

```diff
@@ -1235,3 +1236,114 @@ def nanstd(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     else:
         std = var.dtype.type(np.sqrt(var))
     return std
+
+
+def nancov(m, y=None, rowvar=1, bias=0, ddof=None):
+    """
+    Estimate a covariance matrix, given data, ignoring observations containing
+    NaN values.
+
+    Covariance indicates the level to which two variables vary together.
+    If we examine N-dimensional samples, :math:`X = [x_1, x_2, ... x_N]^T`,
+    then the covariance matrix element :math:`C_{ij}` is the covariance of
+    :math:`x_i` and :math:`x_j`. The element :math:`C_{ii}` is the variance
+    of :math:`x_i`.
+
+    Parameters
+    ----------
+    m : array_like
+        A 1-D or 2-D array containing multiple variables and observations.
+        Each row of `m` represents a variable, and each column a single
+        observation of all those variables. Also see `rowvar` below.
+    y : array_like, optional
+        An additional set of variables and observations. `y` has the same
+        form as that of `m`.
+    rowvar : int, optional
+        If `rowvar` is non-zero (default), then each row represents a
+        variable, with observations in the columns. Otherwise, the relationship
+        is transposed: each column represents a variable, while the rows
+        contain observations.
+    bias : int, optional
```

Comment:
I wonder if we want to keep `bias` and just keep `ddof`? I suppose the argument for keeping it is that `cov` has it. @rkern Thoughts?